maestro-studio/server/src/main/java/maestro/studio/InsightService.kt

```diff
@@ -0,0 +1,58 @@
+package maestro.studio
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.coroutines.*
+import maestro.studio.BannerMessage.*
+import maestro.utils.Insight
+import maestro.utils.Insights
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+
+object InsightService {
+
+    private var currentInsights: List<Insight>? = null
+
+    fun routes(routing: Route) {
+        routing.get("/api/banner-message") {
+            Insights.onInsightsUpdated {
```

Comment:
Move this outside of API callback handler

Reply:
Done ✅